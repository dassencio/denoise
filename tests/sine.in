-0.0816768
0.00520175
0.131854
0.066951
0.107055
0.206326
0.476274
0.267375
0.38222
0.387895
0.728017
0.695729
0.572231
0.704112
0.589955
0.791639
0.788129
0.967704
0.885867
0.886918
0.913659
0.835806
0.882289
0.821747
0.899288
0.892462
1.07554
0.923214
0.953149
0.832936
0.945246
1.03548
0.78452
0.845365
0.921525
0.838221
0.880259
0.685216
0.646212
0.551112
0.488324
0.594564
0.676241
0.379983
0.373701
0.222597
0.043532
0.169933
0.171276
0.114458
-0.10016
-0.177911
-0.218989
-0.180542
-0.0580257
-0.410464
-0.39477
-0.677435
-0.605181
-0.50112
-0.51494
-0.7107
-0.853176
-0.669597
-0.977157
-0.920987
-0.982412
-0.919532
-0.991171
-0.948565
-0.856241
-1.14226
-1.09344
-1.02344
-1.00135
-0.966866
-0.94432
-0.883996
-1.0856
-0.97052
-1.06076
-0.846814
-0.703231
-0.974307
-1.08463
-0.790653
-0.534496
-0.697874
-0.59263
-0.54361
-0.62361
-0.571661
-0.344715
-0.345459
-0.426008
-0.303186
-0.346788
-0.130674
-0.112005
-0.111486
