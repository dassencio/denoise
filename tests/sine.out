0.168624
0.181139
0.202451
0.227293
0.260152
0.300666
0.345897
0.384608
0.429182
0.476104
0.527436
0.568739
0.603692
0.640219
0.67355
0.711062
0.744545
0.775849
0.797559
0.814855
0.828547
0.837984
0.847529
0.855337
0.864824
0.872588
0.879358
0.876319
0.870935
0.861441
0.853371
0.840709
0.818307
0.797595
0.774494
0.744042
0.708881
0.665151
0.620418
0.574394
0.529536
0.486737
0.438548
0.378473
0.318323
0.255405
0.194126
0.140378
0.0851514
0.0256188
-0.0383558
-0.0992401
-0.156191
-0.210002
-0.265286
-0.330933
-0.392603
-0.454165
-0.504563
-0.549931
-0.597739
-0.649687
-0.698585
-0.739753
-0.784428
-0.819467
-0.849431
-0.872745
-0.89372
-0.909822
-0.923987
-0.941539
-0.949056
-0.949353
-0.945946
-0.939768
-0.932236
-0.924099
-0.917968
-0.903455
-0.885588
-0.858963
-0.832946
-0.813414
-0.785838
-0.743322
-0.69844
-0.661755
-0.623263
-0.586304
-0.551479
-0.513048
-0.471686
-0.436672
-0.406219
-0.374777
-0.346914
-0.319058
-0.300621
-0.291614
